submitted_models/marble_spot_sensor_config_1/urdf/basler_cameras.xacro

```diff
@@ -0,0 +1,114 @@
+                <xacro:if value="${fisheye == 0 or '$(arg rendering_target)' == 'ign'}">
+                    <xacro:property name="cam_type" value="camera"/>
+                </xacro:if>
+                <xacro:unless value="${fisheye == 0 or '$(arg rendering_target)' == 'ign'}">
+                    <xacro:property name="cam_type" value="wideanglecamera"/>
+                </xacro:unless>
+                <sensor name="${name}" type="${cam_type}">
+                    <!-- I think the simulated sensor should be placed in the middle of the entrance pupil. -->
+                    <xacro:property name="virtual_lens_origin" value="${lens_back_pos + lens_length - lens_entrance_pupil_position}" />
+                    <pose>${virtual_lens_origin} 0 ${camera_height/2} 0 0 0</pose>
+                    <update_rate>${frame_rate}</update_rate>
+                    <visualize>${visualize}</visualize>
+                    <camera>
+                        <horizontal_fov>${radians(hfov_deg)}</horizontal_fov>
+                        <image>
+                            <width>${resolution_x}</width>
+                            <height>${resolution_y}</height>
+                            <format>R8G8B8</format>
+                        </image>
+                        <clip>
+                            <xacro:property name="clip" value="${lens_entrance_pupil_position * 1.1 if near_clip == -1 else near_clip}" />
+                            <near>${clip}</near>
+                            <far>300</far>
+                        </clip>
+                        <noise>
+                            <type>gaussian</type>
+                            <mean>0.0</mean>
+                            <stddev>0.007</stddev>
+                        </noise>
+                        <xacro:if value="${fisheye and '$(arg rendering_target)' == 'gz'}">
+                            <lens>
+                                <type>stereographic</type>
+                                <scale_to_hfov>true</scale_to_hfov>
+                                <cutoff_angle>${pi}</cutoff_angle>
+                                <env_texture_size>1024</env_texture_size>
+                            </lens>
+                        </xacro:if>
+                    </camera>
+                </sensor>
+            </gazebo>
+        </xacro:if>
+    </xacro:macro>
+
+    <xacro:macro name="basler_ace2_pro_link" params="name hfov_deg fisheye:=0 lens_length lens_back_focal_distance lens_entrance_pupil_position lens_mass lens_mesh near_clip:=-1 simulate:=1 visualize:=0">
+        <xacro:basler_ace_base name="${name}" hfov_deg="${hfov_deg}" frame_rate="9" resolution_x="1920" resolution_y="1200"
+            fisheye="${fisheye}" simulate="${simulate}" visualize="${visualize}"
+            camera_length="0.0555" origin_to_camera_front_dist="0.0524" camera_mass="${basler_ace2_pro_mass}"
+            camera_mesh="package://ctu_cras_norlab_spot_sensor_config_1/meshes/basler_ace2_pro.dae"
```

Comment:
There are a number of references in the `urdf/*xacro` files used to generate the `model.sdf` that carry references to `ctu_cras_norlab_spot_sensor_config_1` for `*.dae` files that are present in this repository.

Reply:
Fixed in https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d